### ISSUE TITLE
Chapter Request: Teresina, Brazil

### ISSUE THREAD

**Issue by thiagocsousa** [papers-we-love/organizers#24]
Where: Teresina, Brazil
Who: Thiago Carvalho de Sousa (@thiagocsousa)

Hi, I'd like to request a repo to start a chapter at my city. We are in Teresina, a northeast city of Brazil. I'm full professor of Computer Science at State University of Piaui and we think that is very important to open this chapter over here, which helps the science development in our region. We already have a meeting about it.

**Comment by hakutsuru** [comment]
@thiagocsousa: I am confused. I created a teresina repo already, and I thought you were involved?

https://github.com/papers-we-love/teresina

The original request was via this issue:

https://github.com/papers-we-love/papers-we-love/issues/418

So there is an existing repo. We hope to add a link for Teresina to the paperswelove.org site when there has been a meeting.

If I have mishandled this, let me know, and the team here will set things right. Thanks.
